app/controllers/concerns/session_path_concern.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module SessionPathConcern
+  extend ActiveSupport::Concern
```

Comment:
This doesn't need to extend `ActiveSupport::Concern` because it's not implementing any concern functionality.

Should these methods actually just be in the `AccountConcern`?

Reply:
The reason I didn't put them in the account concern was so that they can sit apart and be included as a separate module.
The account concern has before filters and so freaks out if you put it in tests for instance.

Perhaps this means it's not really a concern and should be a Plain Old Ruby Module somewhere, but i wasn't sure where that might be...

And I figured for controllers a concern was the place you'd put something that hijacked routes like this?